app/src/views/InstanceView.jsx

```diff
@@ -126,6 +141,7 @@ function InstanceView() {
 					alignItems="center"
 					flex="1"
 				>
+					{/*chunyu to jingwen: what is setIsPopoverOpen? Eslint is giving me an error here.*/}
```

Comment:
oh i think its residual from the congrats screen when i used the popover component prev, can remove any trace of it